corehq/apps/users/static/users/js/roles.js

```diff
@@ -2,8 +2,9 @@ hqDefine('users/js/roles',[
     'jquery',
     'underscore',
     'knockout',
+    'hqwebapp/js/toggles',
     'hqwebapp/js/alert_user',
-], function ($, _, ko, alertUser) {
```

Comment:
is `alertUser` still being used even with the import/requirement above?

Reply:
Added it back https://github.com/dimagi/commcare-hq/pull/31333/commits/98f0929721816b753898de0efc3402c3dde83617